README.md

```diff
@@ -122,6 +122,68 @@ As in the normal mode, print statements are still printed to the terminal.
 
 The pipeline can be run multiple times, as in the beginning of each run the scene is cleared.
 
+### Breakpoint-Debugging in IDEs
+
+As blenderproc runs in blenders separate python environment, debugging your blenderproc script cannot be done in the same way as with any other python script.
+Therefore, remote debugging is necessary, which is explained for vscode and PyCharm in the following:
+
+#### Debugging with vscode
+
+First, install the `debugpy` package in blenders python environment.
+
+```
+blenderproc pip install debugpy
+```
+
+Now add the following configuration to your vscode [launch.json](https://code.visualstudio.com/docs/python/debugging#_initialize-configurations).
+
+```json
+{                        
+    "name": "Attach",
+    "type": "python",
+    "request": "attach",
+    "connect": {
+        "host": "localhost",
+        "port": 5678
+    }
+}
+```
+
+Finally, add the following lines to the top of your blenderproc script which you want to debug.
```

Comment:
But, after the import of blenderproc, right?

Reply:
Yes this should be placed after the imports. I updated the readme